dagrunner/plugin_framework.py

```diff
@@ -248,7 +248,8 @@ def __call__(self, filepath, **kwargs):
         `NodeAwarePlugin`.
 
         Args:
-        - filepath (str): The filepath to be expanded.
+        - `filepath` (str): The filepath to be expanded.
+        - `node_properties`: node properties passed by the plugin executor.
         - **kwargs: Keyword arguments to be used in the expansion.  Node
           properties/attributes are additionally included here as a node aware plugin.
```

Comment:
Comment needs changing now that node properties have their own explicit argument.